photon-colors/package.json

```diff
@@ -20,7 +20,7 @@
   "author": "Blake Winton <bwinton@latte.ca>",
   "license": "MPL-2.0",
   "bugs": {
-    "url": "https://github.com/FirefoxUX/design-tokens/issues"
+    "url": "https://github.com/FirefoxUX/design-tokens/issues?q=is%3Aissue+is%3Aopen+label%3A%22tokens%3A+colors%22"
   },
-  "homepage": "https://github.com/FirefoxUX/design-tokens#readme"
+  "homepage": "https://github.com/FirefoxUX/design-tokens/photon-colors#readme"
```

Comment:
We should probably make these changes in the `photon-components/package.json` file, too.

Reply:
Good one! I also found out that the link needed to have the `/tree/master` path. I added to both `package.json` filed and double-checked all the links.